ecosystem/cli/ci.py

```diff
@@ -17,37 +19,20 @@ class CliCI:
     Ex: `python manager.py ci parser_issue --body="<SOME_MARKDOWN>"`
     """
 
-    def __init__(self, root_path: Optional[str] = None):
-        """CliCI class."""
-        self.current_dir = root_path or os.path.abspath(os.getcwd())
-        self.resources_dir = "{}/ecosystem/resources".format(self.current_dir)
-        self.dao = DAO(path=self.resources_dir)
-        self.logger = logger
-
     @staticmethod
-    def parser_issue(body: str) -> None:
-        """Command for calling body issue parsing function.
+    def add_member_from_issue(body: str, resources_dir: str | None = None) -> None:
```

Comment:
Nit that you can make resources_dir a keyword only arg. Since it's testing only, might want to call it `testing_resources_dir`

```suggestion
    def add_member_from_issue(body: str, *, resources_dir: str | None = None) -> None:
```